nvtabular/ops.py

```diff
@@ -580,6 +582,32 @@ def op_logic(self, gdf: cudf.DataFrame, target_columns: list, stats_context=None
         return z_gdf
 
 
+class Dropna(TransformOperator):
+    """
+    This operation detects missing values, and returns
+    a cudf DataFrame with Null entries dropped from it.
+
+    Although you can directly call methods of this class to
+    transform your categorical features, it's typically used within a
+    Workflow class.
+    """
+
+    default_in = ALL
+    default_out = ALL
+
+    @annotate("Dropna_op", color="darkgreen", domain="nvt_python")
+    def op_logic(self, gdf, target_columns, stats_context=None):
+        cont_names = target_columns
+        if not cont_names:
+            new_gdf = gdf.dropna()
+            new_gdf = new_gdf.reset_index(drop=True)
+            return new_gdf
+        else:
+            new_gdf = gdf.dropna(subset=cont_names)
+            new_gdf = new_gdf.reset_index(drop=True)
+        return new_gdf
```

Comment:
How about merging the if/else block together ? Also we should call inplace to avoid allocating extra memory:

```suggestion
        new_gdf = gdf.dropna(subset=target_columns or None)
        new_gdf.reset_index(drop=True, inplace=True)
        return new_gdf
```